front/src/applications/operationalStudies/components/MacroEditor/NGE.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react';
+
+import ngeMain from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/main.js?url';
+import ngePolyfills from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/polyfills.js?url';
+import ngeRuntime from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/runtime.js?url';
+import ngeStyles from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/styles.css?url';
+import ngeVendor from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/vendor.js?url';
+
+import 'styles/scss/applications/operationalStudies/_nge.scss';
```

Comment:
We don't import directly the css in the application. You can add the file nge in `src/styles/scss/applications/_operationalStudies.scss`

Reply:
Done